sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/src/properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("Microsoft.Communication")]
```

Comment:
Given the alignment with Communication, should this move to the [communication](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/communication) root rather than hanging out here all by itself and lonely?

Reply:
Yes I think that makes sense! I just didn't know that was available! I will move the entire folder over.

Reply:
That took a bit longer than I thought but it's there now!

![image](https://github.com/Azure/azure-sdk-for-net/assets/65766115/9ad35f6f-bc82-445a-86d8-3d2a74708d02)

No longer by itself!